admin/metabox/class-metabox-section-react.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays the React root element for a metabox section.
+ */
+class WPSEO_Metabox_Section_React implements WPSEO_Metabox_Section {
+
+	/**
+	 * Name of the section.
```

Comment:
wouldn't be better to clarify here too this is used as an identifier? currently, it's explained only in the class construct docblock